.github/pull_request_template.md

```diff
@@ -26,8 +26,15 @@ An amazing description should answer some questions like:
 TODO: fill description here
 
 ## How can this PR be tested?
-TODO: fill steps to reproduce here, if applicable,
-      or remove the section
+
+TODO: modify the automated test suite to verify that the PR causes MariaDB to
+behave as intended, using [mysql-test](https://mariadb.com/kb/en/mysql-test-overview).
```

Comment:
Please replace link with "https://mariadb.org/get-involved/getting-started-for-developers/writing-good-test-cases-mariadb-server/". This small tutorial is less fluffy than mtr overview, just a better tool for the job.

Reply:
Thanks, done :heavy_check_mark: